apps/database/supabase/migrations/20260616124500_add_course_tests.sql

```
@@ -0,0 +1,110 @@
+grant update on table "public"."course_test_modules" to "authenticated";
+
+grant delete on table "public"."course_test_modules" to "service_role";
+grant insert on table "public"."course_test_modules" to "service_role";
+grant references on table "public"."course_test_modules" to "service_role";
+grant select on table "public"."course_test_modules" to "service_role";
+grant trigger on table "public"."course_test_modules" to "service_role";
+grant truncate on table "public"."course_test_modules" to "service_role";
+grant update on table "public"."course_test_modules" to "service_role";
+
+-- RLS Policies
+create policy "Allow all access for workspace member"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)